bundle/src/test/java/com/adobe/acs/commons/remoteassets/impl/RemoteAssetDecoratorTest.java

```diff
@@ -19,318 +19,248 @@
  */
```

Comment:
I'm amazed at your ability to modularize code and tests, but I question why the complete rewrite of the work I did here to make this work with an actual sling implementation - i.e. as "real" as possible, using sling mock to have an in-memory JCR.  Though the new test do test functionality on an individual function level, the old tests I feel had an additional value of testing things truly working.  For example, a call to `resourceResolver.getResource()` was doing a real fetch of a real Resource, which was getting decorated by the RemoateAssetDecorator in the natural way it happens in sling.  The new tests have lost all of that.

Were this an original code submission, I'd be completely happy with these tests.  However, given that these tests were already written and in some ways perceivably more valuable, I struggle a bit with these changes.

If all we needed to do was remove PowerMockito, I think your wrapper function around `AccessControlUtil.getUserManager` was already sufficient, as the `remoteAssetDecorator` in the old tests was already a `spy` that can have the new `getUserManager` function mocked.

Reply:
Well, my intention was to just make the build work under `J11`, nothing more. Usually, while reading new code unknown to me, I also apply [The Boy Scout Rule](https://deviq.com/boy-scout-rule/), but after a couple of PRs guys here convinced me to drop it :) I am not sure atm whether it was before or after this PR though.

I've obviously tried the fast and simple approach that you mention, but since it did not work just like that and since it still failed as I had to digg deeper and deeper - I've concluded that a rewrite with dropping the deeper checks approach will simply be much faster. It was simple time economics :) 

So actually I don't care whether the `PowerMockRunner` stays or not and whether my testing approach will replace the existing one or not, as long as the build will pass under `Java 11`...
Do you think you could try and check yourself how to fix your tests as you suggest above? Let me know how I can help.

Reply:
Ok, I'll take a look at this when I get a chance.  How urgent are these java 11 updates?  Will have to squeeze this in somewhere.